source/process/mobile-release-process.md

```diff
@@ -9,58 +9,137 @@ Notes:
 - All cut-off dates are based on 10am (San Francisco Time) on the day stated.
 - T-minus counts are measured in "working days" (weekdays other than major holidays in the US and Canada) prior to release day.
 
-### A. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+### A. (T-minus 12 working days) Cut-off for merging major features
 
-1. PM:
-    - Submit pull request for changelog
+No pull requests for major features should be **merged** to the current release after this date. In special cases, exceptions can be made by the Release Manager.
+
+1. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Draft Changelog in a WIP PR with updates for highlights, feature additions and bug fixes
+    - Start posting a daily Zero Bug Balance query (posted until zero bugs or day of release)
+2. PM:
+    - Prioritize reviewing major features, ensuring any bugs and UX issues get fixed
+    - Check that all major features are behind a feature flag
+    - Review Jira Backlog and move any tickets that will not be merged to the next release
+3. Dev:
+    - Prioritize reviewing, updating, and merging of pull requests for current release until there are no more tickets in the pull request queue marked for the current release
+      - After the cut-off, any PRs that include significant code changes, require approval of the release manager before merging
+4. Marketing:
+    - Prepare a list of highlights to be included in the next platform release announcement
+
+### B. (T-minus 11 working days) Major feature testing
+
+1. QA:
+    - Prioritize testing merged PRs and resolved tickets
```

Comment:
In order to do this we need to cut a beta build. Let's add a step for build at T-12 to do that